Index: src/br/com/dio/PrimerPrograma.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/br/com/dio/PrimerPrograma.java b/src/br/com/dio/PrimerPrograma.java
--- a/src/br/com/dio/PrimerPrograma.java	
+++ b/src/br/com/dio/PrimerPrograma.java	(date 1634156042013)
@@ -2,7 +2,7 @@
 
 public class PrimerPrograma {
     public static void main(String[] args) {
-        
+        System.out.println("Hola mundo");
     }
 
 }
